commit b472bc2eba017b3286dc7110f1c2ce3a039e6ac7
Author: Christian Hesse <mail@eworm.de>
Date:   Tue Jan 2 14:32:21 2018 +0100

    MDEV-17028: Use descriptive file names for sysusers and tmpfiles configuration
    
    These files were installed to:
    
    ${INSTALL_SYSTEMD_SYSUSERSDIR}/sysusers.conf
    ${INSTALL_SYSTEMD_TMPFILESDIR}/tmpfiles.conf
    
    Instead rename the files to more descriptive file names 'mariadb.conf'.

diff --git a/support-files/CMakeLists.txt b/support-files/CMakeLists.txt
index 62ad8bcffc9..b3162f484ce 100644
--- a/support-files/CMakeLists.txt
+++ b/support-files/CMakeLists.txt
@@ -118,7 +118,8 @@ IF(UNIX)
       CONFIGURE_FILE(sysusers.conf.in
               ${CMAKE_CURRENT_BINARY_DIR}/sysusers.conf @ONLY)
       INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/sysusers.conf
-              DESTINATION ${INSTALL_SYSTEMD_SYSUSERSDIR} COMPONENT Server)
+              DESTINATION ${INSTALL_SYSTEMD_SYSUSERSDIR}
+              RENAME mariadb.conf COMPONENT Server)
     ENDIF()
 
     IF(INSTALL_SYSTEMD_TMPFILESDIR)
@@ -126,7 +127,8 @@ IF(UNIX)
       CONFIGURE_FILE(tmpfiles.conf.in
               ${CMAKE_CURRENT_BINARY_DIR}/tmpfiles.conf @ONLY)
       INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/tmpfiles.conf
-              DESTINATION ${INSTALL_SYSTEMD_TMPFILESDIR} COMPONENT Server)
+              DESTINATION ${INSTALL_SYSTEMD_TMPFILESDIR}
+              RENAME mariadb.conf COMPONENT Server)
     ENDIF()
 
     # @ in directory name broken between CMake version 2.8.12.2 and 3.3
